modules/dhcp_common/server.rb

```diff
@@ -118,44 +118,42 @@ def inspect
     def add_record options = {}
       related_macs = options.delete("related_macs") || []
       logger.debug "Ignoring duplicates for macs: #{related_macs.inspect}" unless related_macs.empty?
-      options = clean_up_add_record_parameters(options)
+      name, ip_address, mac_address, subnet_address, options = clean_up_add_record_parameters(options)
 
-      validate_ip(options[:ip])
-      validate_mac(options[:mac])
-      raise(Proxy::DHCP::Error, "Must provide hostname") unless options[:hostname]
```

Comment:
Is this safe to remove because we use "name" as a fallback? Should we add a check if name is set?